docs/UserGuide.md

```diff
@@ -225,12 +225,12 @@ Add a link for lecture/tutorial sessions through zoom
 Format: `add link m/<module code> <compulsory arguments>`
 
 List of `<compulsory arguments>`:
-- `t/<TYPE>` type of lesson (`lecture`, `tutorial`, `lab`, `project`).
+- `t/<TYPE>` type of usa (`lecture`, `tutorial`, `lab`, `project`).
```

Comment:
sorry what do u mean by *usa*?

Reply:
oh! I will fix this typo. It should be "use" since I think "project" doesn't sound like a lesson.